### DISCUSSION TITLE
Links on record history

### DISCUSSION THREAD

**Discussion by InitiativeWorking** [burkybang/SuiteAdvanced-Feedback#112] [Ideas]
I think it'd be very useful to have the links to the users who made changes in a record or the workflows and their states that did the changes in the [record]>System Information>System Notes/Active Workflows/Workflow History lists so we can jump directly when debugging.

**Comment by burkybang** [comment]
Keep the ideas coming. I love them.

This is definitely a useful one. I'll think about how I can make this possible.
